utils/doclint/xmlDocumentation.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// @ts-check
+
+const Documentation = require('./documentation');
+const { visitAll } = require('../markdown');
+/**
+ * @param {Documentation.MarkdownNode[]} nodes
+ * @param {number} maxColumns
+ */
+function renderXmlDoc(nodes, maxColumns) {
+    const summary = [];
+    const examples = [];
+    /** @type {Documentation.MarkdownNode} */
+    let lastNode;
+
+    summary.push('<summary>');
+
+    visitAll(nodes, (node, depth) => {
+        lastNode = innerRenderXmlNode(node, lastNode, summary, examples, maxColumns);
+    });
+
+    if (summary.length == 1) { // just the <summary> node 
```

Comment:
Why do we need to handle this edge case?

Reply:
It's supposed to be a _cheap_ way of getting rid of:

```csharp
    /// <summary>
    /// </summary>
```

but instead of ignoring it, I might be better off outputting a warning as that means it's not documented.